### ISSUE TITLE
KDE / Plasma 5: system tray icon no longer appears since 0.9.61

### ISSUE THREAD

**Issue by igalic** [Fushko/gammy#76]
starting gammy in plasma 5:

```
meena@beastix ~> gammy
2021-01-12 20:19:51.369 DEBUG [100583] [config::read@92] Reading from: /home/meena/.config/gammyconf
2021-01-12 20:19:51.466 ERROR [100583] [MainWindow::listenWakeupSignal@148] Wakeup interface not found.
2021-01-12 20:19:51.466 ERROR [100583] [MainWindow::init@28] Gammy is unable to reset the proper brightness / temperature when resuming from suspend.
QObject::startTimer: Timers can only be used with threads started with QThread
QObject::startTimer: Timers can only be used with threads started with QThread
QObject::startTimer: Timers can only be used with threads started with QThread
QObject::startTimer: Timers can only be used with threads started with QThread
```

pulls the screen into red, but there's no GUI and no system tray icon.
This seems to be a regression with the 0.9.61 update.

**Comment by Fushko** [comment]
Damn, thought it was only gnome. Referencing #57

Btw did these also happen previously?
2021-01-12 20:19:51.466 ERROR [100583] [MainWindow::listenWakeupSignal@148] Wakeup interface not found.
2021-01-12 20:19:51.466 ERROR [100583] [MainWindow::init@28] Gammy is unable to reset the proper brightness / temperature when resuming from suspend.

**Comment by Fushko** [comment]
Fixed on `develop`

**Comment by igalic** [comment]
> Btw did these also happen previously?
> 
> ```
> 2021-01-12 20:19:51.466 ERROR [100583] [MainWindow::listenWakeupSignal@148] Wakeup interface not found.
> 2021-01-12 20:19:51.466 ERROR [100583] [MainWindow::init@28] Gammy is unable to reset the proper brightness / temperature when resuming from suspend.
> ```

i believe so, yes.
i thought that enabling dbus would fix this,but all it fixed was that gammy stopped complaining about not being able to find dbus, and hence would be unable to do those things that it's still complaining about.

i don't mind too much, sleep and resume don't work very well on FreeBSD

**Comment by grahamperrin** [comment]
Not just the tray icon; I don't get a window when starting the application. 

FreeBSD bug [252599 – accessibility/gammy no longer presents its GUI](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=252599)

Below, I did not start Gammy after starting KDE but I find two instances running: 

![2021-01-13 08:29](https://user-images.githubusercontent.com/192271/104426672-24667e80-557a-11eb-89a5-2aa7033e1ea2.png)

– I'll kill the second instance. 

I assume that (in the absence of a GUI) I inadvertently ran Gammy twice in a previous KDE session, and that the doubling-up was respected by session restoration.

**Comment by Fushko** [comment]
Should be fixed in 0.9.62. @igalic let me know.
@grahamperrin multiple instances are also prevented now.

**Comment by igalic** [comment]
https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=252692 update is in progress

**Comment by grahamperrin** [comment]
Fix confirmed in `0.9.62` on FreeBSD-CURRENT. Thank you!
